src/components/Profile.tsx

```diff
@@ -79,7 +82,8 @@ export function Profile(): JSX.Element {
   const { user } = useData();
   const [copied, setCopied] = useState<Identifier>("none");
   const [qrCodeIdentifier, setQrCodeIdentifier] = useState<Identifier>("none");
-  if (!isUserLoggedIn(user)) {
+  const isLoggedIn = isUserLoggedIn(user) || isUserLoggedInWithExtension(user);
```

Comment:
I see this a lot through the PR and it's easy to miss, how about:

- rename isUserLoggedIn to isUserLoggedInWithBrowser
- and create a new isUserLoggedIn { return isUserLogedInWithBroser(user) || isUserLoggedInWithExtension}

Reply:
will do, but i'll rename it to isUserLoggedInWithSeed since i'd see a login with a browser-extension more as a loginWithBrowser